CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java

```diff
@@ -187,19 +190,20 @@ private static class JsonEntity {
      * files in this array will be executed as integration tests.
      */
     private static final String[] JSON_TEST_FILES = {
-       "tests/example_use_cases_doc/1a.json",
-       "tests/example_use_cases_doc/2a.json",
-       "tests/example_use_cases_doc/2b.json",
-       "tests/example_use_cases_doc/2c.json",
-       "tests/example_use_cases_doc/3a.json",
-       "tests/example_use_cases_doc/3b.json",
-       "tests/example_use_cases_doc/3c.json",
-       "tests/example_use_cases_doc/4a.json",
-       // "tests/example_use_cases_doc/4c.json", // currently disabled because it uses action attributes
-       "tests/example_use_cases_doc/4d.json",
-       "tests/example_use_cases_doc/4e.json",
-       "tests/example_use_cases_doc/4f.json",
-       "tests/example_use_cases_doc/5b.json",
+       "tests/decimal/1.json",
+       "tests/decimal/2.json",
+       "tests/example_use_cases/1a.json",
+       "tests/example_use_cases/2a.json",
+       "tests/example_use_cases/2b.json",
+       "tests/example_use_cases/2c.json",
+       "tests/example_use_cases/3a.json",
+       "tests/example_use_cases/3b.json",
+       "tests/example_use_cases/3c.json",
```

Comment:
[nit] Can we keep the comment explaining why 4c isn't run?

Reply:
I deleted it because that test was not moved over to https://github.com/cedar-policy/cedar-integration-tests -- do you think we should maintain this test in our integration test repo even if it's not used?